README.md

```diff
@@ -168,7 +168,14 @@ The raw demonstration files contain all necessary information (e.g. initial stat
 ```bash
 # Replay demonstrations with control_mode=pd_joint_delta_pos
 python tools/replay_trajectory.py --traj-path demos/rigid_body_envs/PickCube-v0/trajectory.h5 \
-  --save-traj --target-control-mode pd_joint_delta_pos --num-procs 10
+  --save-traj --target-control-mode pd_joint_delta_pos --obs-mode none --num-procs 10
+  
+# For soft-body environments, if you haven't used an environment in the past and 
```

Comment:
Please move the comment somewhere. For example, adding a ">" (markdown quote) paragraph below the code block.

Maybe we can remove the toggle list for warp installation. Fanbo has already provided a script to precompile SDF for soft-body environments, and we can link that.